docs/rst/user_manual/configuration.rst

```diff
@@ -0,0 +1,300 @@
+The yaml must contain a tag ``allowlist`` which element is a vector (``[]``).
+This vector elements are the different Topics that will be transmitted (this requires to create the
+:term:`Writers<DataWriter>` and :term:`Readers<DataReader>` for each topic for each Participant).
+See Topic section for further information about the topic
+
+.. todo:
+
+    Add link to topic page when created
+
+In the following yaml example, the |ddsrouter| will transmit the topic ``rt/chatter`` (default ROS2 topic for
+``talker`` and ``listener``) with type name ``std_msgs::msg::dds_::String_``.
+It also will transmit the topic  ``HelloWorldTopic`` (default FastDDS topic for ``HelloWorldExample``)
+with type name ``HelloWorld``.
+
+.. code-block:: yaml
+
+    allowlist:
+    [
+        {name: "rt/chatter", type: "std_msgs::msg::dds_::String_"},
+        {name: "HelloWorldTopic", type: "HelloWorld"},
+    ]
+
+.. note::
+
+    Tag ``allowlist`` must be at yaml base level (it must not be inside any other tag).
+
+
+Participant Configuration
+=========================
+
+At the yaml base level, along with ``allowlist`` tag, there will be the different Participants that will be created,
+along with their specific configuration.
+Each Participant is identified by a unique :term:`Participant Id` that will be the yaml *key*.
+The yaml *value* for this *key* is the configuration for this specific Participant.
+There could be any number of Participants, and Participant types could be repeated.
+
+Each Participant has its specific configuration.
+Please, refer to :ref:`user_manual_participant_participant_types` in order to see each of the
+:term:`Participant Types<Participant Type>` requirements.
+
+.. warning::
+
+    Do not configure two Participants in a way that they can communicate to each other (e.g. two Simple participants
+    in the same domain).
+    This will lead to an infinite feedback loop for one to each other.
+
+In the following yaml example, the |ddsrouter| will create two Simple Participants, one for ``domain 0`` and one for
+``domain 1``.
+This is a typical use case of Domain bridge.
+The topics allowed in the two domains will start communicating to each other .
+Be aware that the communication is not P2P, the data must reach the |ddsrouter| and this will forward the data.
+
+.. todo:
+
+    Add link to Simple Participant when page created
+    Add link to use case domain bridge
+
+.. code-block:: yaml
+
+    Participant0:       # Participant Id = Participant0
+        type: local     # Participant Type = simple
+        domain: 3       # DomainId = 3
+
+    ################
+
+    simple:             # Participant Id = simple ; Participant Type = simple
+        domain: 6       # DomainId = 6
+
+The first Participant `Participant0` has Participant Id *Participant0* and Participant Type *simple*.
+The second Participant has Participant Id *simple*, and type is not required to be specified as it is get from the
```

Comment:
```suggestion
The second Participant has Participant Id *simple* and configured to be of the *simple* type. That is because the type is not required to be specified as it is get from the
```

Reply:
:+1: